config/monitoring/prometheus/apps/prometheus-configs.yaml

```diff
@@ -1692,3 +1717,101 @@ data:
           labels:
             severity: warning
             namespace: redhat-ods-applications
+
+  feast-operator-recording.rules: |
+    groups:
+      - name: SLOs - Feast Operator Controller Manager
+        rules:
+        - expr: |
+            absent(up{job="feast-operator"}) * 0 or vector(1)
+          labels:
+            instance: feast-operator-controller-manager
+          record: probe_success
+    
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="feast-operator-controller-manager"}[1d]))
+          labels:
+            instance: feast-operator-controller-manager
+          record: probe_success:burnrate1d
+    
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="feast-operator-controller-manager"}[1h]))
+          labels:
+            instance: feast-operator-controller-manager
+          record: probe_success:burnrate1h
+    
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="feast-operator-controller-manager"}[2h]))
+          labels:
+            instance: feast-operator-controller-manager
+          record: probe_success:burnrate2h
+    
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="feast-operator-controller-manager"}[30m]))
+          labels:
+            instance: feast-operator-controller-manager
+          record: probe_success:burnrate30m
+    
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="feast-operator-controller-manager"}[3d]))
+          labels:
+            instance: feast-operator-controller-manager
+          record: probe_success:burnrate3d
+    
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="feast-operator-controller-manager"}[5m]))
+          labels:
+            instance: feast-operator-controller-manager
+          record: probe_success:burnrate5m
+    
+        - expr: |
+            1 - min(avg_over_time(probe_success{instance="feast-operator-controller-manager"}[6h]))
+          labels:
+            instance: feast-operator-controller-manager
+          record: probe_success:burnrate6h
+
+  feast-operator-alerting.rules: |
+    groups:
+      - name: SLOs-probe_success_feast_operator
+        rules:
+          - alert: Feast Operator Probe Success Burn Rate
+            annotations:
+              message: 'High error budget burn for {{ $labels.instance }} (current value: {{ $value }}).'
+              triage: "https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Model-Serving/rhoai-feast-operator-probe-success-burn-rate.md"
```

Comment:
we will need this to be added as well and i do not think  you should put yours in "https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Model-Serving" but create a new folder

Reply:
@zdtsw 
updated to use it from https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Feature-Store/rhoai-feast-operator-probe-success-burn-rate.md

Also do you know how fix the unit test,  I think I need to configure generate / the file `feast-operator-alerting.rules.yaml` `  WARNING: no file match pattern tests/prometheus_unit_tests/
  FAILED:
`

Reply:
yeah, you will need a file in ./tests/prometheus_unit_tests/ for the promtest as well

Reply:
how this will be generated, for example if run `make test-alerts` the file generated `feastoperator-alerting.rules.yaml` but it's empty. I believe I missing some steps here.

Reply:
you will need to generate this file and put the test in it, 

make test-alerts only does "promtool test rules *.rules.yaml"

Reply:
Thanks, updated accordingly.